server/models/blogs.js

```diff
@@ -0,0 +1,34 @@
+import mongoose from 'mongoose';
+
+const { Schema } = mongoose;
+
+const BlogsSchema = new Schema({
+  userId: {
+    type: Schema.Types.ObjectId,
+    ref: 'User',
+  },
+  title: {
+    type: String,
+    required: true,
+  },
+  content: [
+    {
+      _id: Schema.Types.ObjectId,
+      type: Schema.Types.Mixed,
+      content: {
```

Comment:
Content within content? A better key nMe can be used here. Also why not create another content collection and then use ref ?

Reply:
1. Better name assigned to inner `content`.
2. The `contentFragment` object represents tagged fragments of a blog post. For example, each paragraph is a different fragment, each image will be its own fragment, similarly for each video file.This is being done to make displaying rich media content in the blog posts easier. 
In this respect, having the contentFragment object as a document outside the `content` array or the BlogSchema document does not make a lot of sense.